bgpd/bgp_routemap.c

```diff
@@ -4400,7 +4400,7 @@ DEFUN (no_set_distance,
 
 DEFUN (set_local_pref,
        set_local_pref_cmd,
-       "set local-preference (0-4294967295)",
+       "set local-preference WORD",
```

Comment:
What is the purpose of passing minus sign for local-pref?

Reply:
For traffic engineering? I mean ISPs use this for lowering local-preference to prefer another route by receiving an arbitrary community.

Reply:
Thanks. I see that in function route_value_compile it is taking care of range as well. 

if (*arg == 0 || *endptr != 0 || errno || larg > UINT32_MAX)

Code looks good to me. Just a thought do we need to provide help here to hint range to user?

Reply:
I guess that would be useful. I'll update.

Reply:
@Spantik updated.